README.md

```diff
@@ -33,19 +33,19 @@ No more compromise on performance, customization, and scalability.
     <br />
     <a href="https://discord.com/invite/SJyz2VJGxV"><strong>  🫱🏼‍🫲🏼 Discord support </strong></a>
     <br />
-    <a href="https://docs.taipy.io/en/latest/gallery/"><strong> 👀 Demos & Examples </strong></a>
+    <a href="https://docs.taipy.io/en/latest/gallery/"><strong> 👀Want some Demos & Examples </strong></a>
   </p>
 
 &nbsp;
 
-## ⭐️ What's Taipy?
+## ⭐️ What's Taipy🤔?
 
 Taipy is designed for data scientists and machine learning engineers to build data & AI web applications.
 &nbsp;
 
-⭐️ Enables building production-ready web applications. <br />
-⭐️ No need to learn new languages. Only Python is needed.<br />
-⭐️ Concentrate on Data and AI algorithms without development and deployment complexities.<br />
+⭐️->Enables building production-ready web applications. <br />
+⭐️-> No need to learn new languages. Only Python is needed.<br />
+⭐️-> Concentrate on Data and AI algorithms without development and deployment complexities.<br />
```

Comment:
The ⭐️ is already acting like a bullet. I don't understand the purpose of the "->". Can you explain?